src/cascadia/WindowsTerminal/pch.h

```diff
@@ -29,6 +29,7 @@ Module Name:
 #include <cstring>
 #include <shellscalingapi.h>
 #include <windowsx.h>
+#include <ShObjIdl.h>

```

Comment:
Do we really want to add this to `pch.h`? Personally I try to avoid adding things into PCH because the damage it does to compiling speed.

Reply:
meh, this one's not so bad. We'll probably end up using it in other places across the project, and I'd rather eat the compile time _once_ in the pch phase rather than many times across rebuilds. 

It's the winrt headers that _really_ chew up pch time 😑